packages/account/src/configs.ts

```diff
@@ -1,4 +1,9 @@
+/**
+ * @deprecated This export is deprecated and will be removed in the future.
+ */
 export const DEVNET_NETWORK_URL = 'https://devnet.fuel.network/v1/graphql';
+
+/**
+ * @deprecated This export is deprecated and will be removed in the future.
+ */
```

Comment:
@Dhaiwat10 Could we replace these constants on the code snippets in the docs with hardcoded strings for each net? If we're deprecating them, we should stop suggesting their use.

Off the top of my head, I remember these two, but there can be more:
- https://docs.fuel.network/docs/fuels-ts/getting-started/connecting-to-a-local-node/
- https://docs.fuel.network/docs/fuels-ts/getting-started/connecting-to-testnet/

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3107/commits/d548512f5dfe2376f1aec0cf90ec123349297008